dwave/system/coupling_groups.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import dwave_networkx as dnx
+
+
+def coupling_groups(hardware_graph):
```

Comment:
✔️

Reply:
Was the `docs/reference/utilities.rst` location  rather than the proposed `reference/embedding Utilities` intentional? The first is a bit of an odd duck in this repo?

Reply:
It was intentional.  I see `docs/reference/utilities.rst` as a better fit because (a) `coupling_groups` is a utility function, in plain terms (despite not being in the `utilities` module) and (b) it is not related to embedding.

Is there is a better location or better way to do this?  We could rename `utilities.rst` to `miscellaneous.rst` and make `Coupling Groups` and `Utilities` sections in it, for example.